src/kernels/jupyter/serverSelector.ts

```diff
@@ -587,12 +588,16 @@ class JupyterServerSelector_Original implements IJupyterServerSelector {
                 if (!handled) {
                     return DataScience.jupyterSelfCertExpiredErrorMessageOnly();
                 }
-            } else if (!this.isWebExtension) {
-                return DataScience.remoteJupyterConnectionFailedWithoutServerWithError().format(
-                    err.message || err.toString()
-                );
             } else {
-                return DataScience.remoteJupyterConnectionFailedWithoutServerWithErrorWeb().format(
+                // Notify the user of web extension specific connection help
```

Comment:
We have two different server selection flows currently. This is the original flow that uses an input box. For this scenario keeping the validation returned is nice. So I broke this up into a smaller specific error message for the web specific connection error info, then the normal connection error message that shows in the red validation box.